usage/rest-server/src/test/java/brooklyn/rest/resources/ServerResourceTest.java

```diff
@@ -97,5 +99,15 @@ public void testShutdown() throws Exception {
                 assertFalse(getManagementContext().isRunning());
             }});
     }
+
+    @Test
+    void testGetConfig() throws Exception {
+        ((BrooklynProperties)getManagementContext().getConfig()).addFrom(ImmutableMap.of("foo.bar.baz", "quux"));
```

Comment:
Better to do `((ManagementContextInternal)getManagementContext()).getBrooklynProperties()`. It effectively does the same thing, but we know we can always cast to the "internal" version (whereas casting the return value of a getter isn't guaranteed moving forwards).

Also, you can use `.put("foo.bar.baz", "quux")` rather than needing the map.